spec/ysh-funcs-builtin.test.sh

```diff
@@ -104,17 +104,17 @@ false
 #### sum()
 var start = 42
 
-write $[sum( range(3) )]
-write $[sum( range(3), start)]
-write $[sum( range(0), start)]
+write $[sum( 0:3 )]
+write $[sum( 0:3, start)]
+write $[sum( 0:0, start)]
 ## STDOUT:
 3
 45
 42
 ## END
 
 #### reversed()
```

Comment:
Actually I think we should specify that reversed() only works on lists --  I don't see a reason to make it work on ranges

That was sort of a Python legacy